Commands/gemini3.py

```
@@ -126,33 +237,38 @@ def reply_with_grounded_gemini(self, message):
-    }, model)
+        result = gemini_generate({
+            "prompt": prompt,
+            "grounded": is_grounded,
+            "grounding_text": grounding_text
+        }, model)
 
-    if "Error" in result[0]:
-        self.send_privmsg(cmd.channel, f"Failed to generate a response. Please try again later.")
-        return
+        if "Error" in result[0]:
+            self.send_privmsg(cmd.channel, f"Failed to generate a response. Please try again later.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)